src/main/scala/com/gd/twitteranalytics/Tweet.scala

```diff
@@ -1,5 +1,5 @@
 package com.gd.twitteranalytics
 
-import java.sql.Timestamp
+import java.sql.Date
```

Comment:
As last discussed, we should use DataSet[Tweet] instead of Dataframe. So, when I try to convert my DataFrame with Column of type java.util.Date to DataSet, I get this error 
java.lang.UnsupportedOperationException: No Encoder found for java.util.Date
The issue is, Spark SQL DataType mapping.

Reply:
oh.. ok, confused it with Timestamp.